packages/discovery-react-components/src/components/DocumentPreview/utils/__tests__/documentData.test.ts

```diff
@@ -35,6 +42,16 @@ describe('documentData', () => {
     expect(consoleError).not.toHaveBeenCalled();
   });
 
+  it('returns correct text mapping data', () => {
+    const mappings = getTextMappings(correctTextMapping)!;
+    expect(mappings).not.toEqual(null);
+    expect(mappings.pages[0].page_number).toEqual(0);
+    expect(mappings.pages[0].height).toEqual(792);
+    expect(mappings.pages[0].width).toEqual(612);
+    expect(mappings.pages[0].origin).toEqual('TopLeft');
```

Comment:
I think you can do a comparison against an actual object, rather than having to do each property individually.